### ISSUE TITLE
Install dialog doesn't meet WCAG 1.4.10 reflow requirement

### ISSUE THREAD

**Issue by SteveL-MSFT** [wixtoolset/issues#6584]
## Bugs

If this issue is a bug:

* Which version of WiX are you building with?

> 3.1.1

* Which version of Visual Studio are you building with (if any)?

> N/A

* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?

> N/A

* Which version of .NET are you building with?

> .NET 6 RC1

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?

> Win 10.0.19043

* Describe the problem and the steps to reproduce it.

> Install https://github.com/PowerShell/PowerShell/releases/download/v7.1.4/PowerShell-7.1.4-win-x64.msi

* Describe the behavior you expected and how it differed from the actual behavior.

> https://www.w3.org/TR/WCAG21/#reflow Accessibility requirement is that 320x256 resolution does not require scrolling in both directions

![image](https://user-images.githubusercontent.com/11859881/134077878-93134703-99e3-4b4f-9e84-32c9ea88e6a5.png)

**Comment by barnson** [comment]
Installer dialogs aren't web content and MSI doesn't support flowable UI.
